conf/application.conf

```diff
@@ -67,6 +67,9 @@ logger.application=DEBUG
 # Email address of site administrator
 hub.admin.email="rrodgers@mit.edu"
 
+# URL of indexing service
+hub.index.url="http://localhost:9200/scoap3/"
```

Comment:
Will this break on Heroku if we merge to master and it deploys as that won't exist? Maybe we need to switch to an ENV var for this before we merge? Or maybe the Indexer doesn't care if it fails at which point that's good for now (and horrible for later when we want to know it hasn't indexed something).